azext_iot/tests/test_iot_central_unit.py

```diff
@@ -169,3 +173,56 @@ class TestMonitorEvents:
     def test_monitor_events_invalid_args(self, timeout, exception, fixture_cmd):
         with pytest.raises(exception):
             subject.iot_central_monitor_events(fixture_cmd, app_id, timeout=timeout)
+
+
+@pytest.mark.skipif(
+    not validate_min_python_version(3, 5, exit_on_fail=False),
+    reason="minimum python version not satisfied",
```

Comment:
We can remove the validate_min_python_version checks now since min 3.5 is assumed.